kolibri/plugins/setup_wizard/assets/src/views/importFacility/SelectFacilityForm.vue

```diff
@@ -143,7 +143,12 @@
         const $credentialsForm = this.$refs.credentialsForm;
         if ($credentialsForm) {
           // The form makes the call to the startpeerfacilityimport endpoint
-          return $credentialsForm.startImport();
+          return $credentialsForm.startImport().then(() => {
```

Comment:
At this point `POST http://localhost:8000/api/tasks/facilitytasks/startpeerfacilityimport/` returns 401 when I pass bad credentials:

`{"detail":"401 Client Error: Unauthorized for url: https://kolibri-beta.learningequality.org/api/morango/v1/certificates/"}`

Then it just gets stuck and I have to refresh the page.

![screenshot-localhost_8000-2020 07 21-12_13_33](https://user-images.githubusercontent.com/6356129/88096531-a33ea980-cb4b-11ea-9084-2640be6f42dc.png)

Reply:
Cool, I found what was causing this 👍